core/domain/state_domain.py

```diff
@@ -1406,7 +1406,11 @@ def validate(self, exp_param_specs_dict, allow_null_interaction):
         self.recorded_voiceovers.validate(content_id_list)
 
     def get_training_data(self):
-        """Retrieves training data from the State domain object."""
+        """Retrieves training data from the State domain object.
```

Comment:
@puneeth2001, This is a duplicate PR for the same issue (You can find first one over [here](https://github.com/oppia/oppia/pull/7741/files#diff-fddf42bff012423be27edc64edd9e4e9R1425).)

Would you mind picking up a different part of the same issue?

Reply:
No Problem @DubeySandeep can you assign me, on which one should I work.

Reply:
@puneeth2001, I've assigned you to another part of the same issue, please check the issue thread. :)